From d987b0dfd3c901336547887105d0bc14b6458a73 Mon Sep 17 00:00:00 2001
From: "Wladimir J. van der Laan" <laanwj@gmail.com>
Date: Sun, 15 Sep 2013 18:52:38 +0200
Subject: [PATCH] include: EGL: always imply MESA_EGL_NO_X11_HEADERS

As we don't use X11 on GCW Zero, this avoids people having to
define MESA_EGL_NO_X11_HEADERS when compiling games.
---
 include/EGL/eglplatform.h |   14 --------------
 1 file changed, 14 deletions(-)

diff --git a/include/EGL/eglplatform.h b/include/EGL/eglplatform.h
index 21b18fe..c8ccb85 100644
--- a/include/EGL/eglplatform.h
+++ b/include/EGL/eglplatform.h
@@ -106,25 +106,11 @@ typedef void                        *EGLNativeDisplayType;
 
 #elif defined(__unix__)
 
-#ifdef MESA_EGL_NO_X11_HEADERS
-
 typedef void            *EGLNativeDisplayType;
 typedef khronos_uintptr_t EGLNativePixmapType;
 typedef khronos_uintptr_t EGLNativeWindowType;
 
 #else
-
-/* X11 (tentative)  */
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
-
-typedef Display *EGLNativeDisplayType;
-typedef Pixmap   EGLNativePixmapType;
-typedef Window   EGLNativeWindowType;
-
-#endif /* MESA_EGL_NO_X11_HEADERS */
-
-#else
 #error "Platform not recognized"
 #endif
 
-- 
1.7.9.5

